### PR TITLE
Issue #KN-0000 feat: Bootcamp setup changes

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,103 +1,169 @@
...
-        proxyReqOpts.headers['Content-Type'] = 'application/json';
-        proxyReqOpts.headers['user-id'] = 'content-editor';
-        proxyReqOpts.headers['Cookie'] = PORTAL_COOKIES;
-        proxyReqOpts.headers['authorization'] = `Bearer ${API_AUTH_TOKEN}`;
+app.use(express.static(__dirname + '/dist/sunbird-collection-editor/'));
+
+var publicRequestHeaders = {
+    authorization: `Bearer ${API_AUTH_TOKEN}`,
+    "x-authenticated-user-token": USER_TOKEN,
+    "x-channel-id": '01374279726315929680',
```
Comment (vinukumar-vs) [inline server.js:25]: Move to env with the default channel (bootcamp).

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

**Review by vinukumar-vs — APPROVED** [review]

File: src/app/data.ts [hunk 2]
```diff
@@ -1,1184 +1,122 @@
...
-        children: {
-          Content: [
-            'Explanation Content',
-            'Learning Resource',
-            'eTextbook',
-            'Teacher Resource',
-            'Course Assessment'
-          ]
-        }
-      },
```
Comment (vinukumar-vs) [inline src/app/data.ts:103]: comment out the code rather than removing the code

File: src/app/app.component.html [hunk 3]
```diff
@@ -2,11 +2,5 @@
     <lib-editor [editorConfig]="editorConfig" (editorEmitter)="editorEventListener($event)"></lib-editor>
 </ng-container>
 <div *ngIf="!editor" class="ui container d-flex flex-jc-center flex-ai-center p-20">
-    <button class="sb-btn sb-btn-normal sb-btn-outline-primary mr-10 font-weight-bold fs-1" (click)="setType('question')" >Question Editor</button>
```
Comment (vinukumar-vs) [inline src/app/app.component.html:5]: Disable these buttons rather than removing

File: server.js [hunk 4]
```diff
@@ -1,103 +1,169 @@
...
+}
+
+const customDecorateReqHeaders = function () {
+    return function (proxyReqOpts, srcReq) {
+        proxyReqOpts.headers = Object.assign({}, proxyReqOpts.headers, publicRequestHeaders);
+        return proxyReqOpts;
+    }
+}
+
+app.post(['/action/content/v3/upload/url/*'], proxy(BASE_URL, {
```
Comment (vinukumar-vs) [inline server.js:47]: Why did we onboarded these API's now. How it was working before?

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (vaibhavbhuva) [reply server.js]: Resolved.

File: src/app/app.component.html (hunk 3 above)
Comment (vaibhavbhuva) [reply src/app/app.component.html:5]: I have updated the code. Please check the same

File: server.js (hunk 4 above)
Comment (vaibhavbhuva) [reply server.js:59]: Before It was not working locally so we have fixed all the APIs in order to work locally.

**Review by pallakartheekreddy — APPROVED** [review]

**Pull request merged**